### PR TITLE
Resource and Design Updates

### PR DESCRIPTION
**Pull request by arfazca**
- [x] Updated ECSS Logo (from ESS Logo)
- [x] Updated Textual Description "We are a community of student developers, designers, and entrepreneurs who are passionate about building software solutions for students at University of Victoria"
- [x] Added VikeLabs Logo in the Header
- [x] Updated Gradient for Projects page

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const About: NextPage = () => {
         <Container maxW="container.xl">
           <SimpleGrid columns={[1, 1, 2]} spacing={10}>
             <Heading color="white">
-              We're part of the UVic Engineering Student Society
+              We're part of the UVic Engineering and Computer Science Students' Society
             </Heading>
             <Text color="white">
               We get funding from the UVic Engineering Student Society to
```
Comment (AmandaGAnderson) [inline pages/about.tsx]: This is also using the outdated ECSS name

File: pages/index.tsx [hunk 2]
```diff
@@ -50,9 +50,9 @@ export const Index: NextPage = () => {
                 Connect; Learn; Build;
               </Heading>
               <Text fontSize="1em" color="gray.700" my="5">
-                We are a community of student developers, designers, and
-                entrepreneurs who are passionate about building the future of
-                the software.
+                We are a community of student developers, designers, and entrepreneurs
```
Comment (AmandaGAnderson) [inline pages/index.tsx]: The other description I've seen used for VikeLabs is, "VikeLabs is a collective of students who learn to build, deploy and test software apps quickly. The group views UVic as a kind of laboratory for testing solutions to problems that exist within the campus community." 

@liamshatzel do you have a preference for which we use on the website? Or we could try a combo of that one and the one currently on the website?

Also if we keep the change currently in the PR, could we please end the description with a period?

**Review by AmandaGAnderson — CHANGES_REQUESTED** [review]
A couple of suggestions/discussions in other comments that should be resovled first. And #99 should be completed first because this duplicates the changes.

File: pages/index.tsx (hunk 2 above)
Comment (AmandaGAnderson) [reply pages/index.tsx]: An option could be, "VikeLabs is a collective of students who learn to build, deploy, and test software apps. We are a community of student developers, designers, and entrepreneurs who are passionate about designing software solutions for students and the UVic campus community."

Comment (liamshatzel) [reply pages/index.tsx]: Yeah lets use that one that Amanda wrote out

File: pages/about.tsx (hunk 1 above)
Comment (arfazca) [reply pages/about.tsx]: @AmandaGAnderson According to [ECSS's](https://www.uvicecss.ca/) website, it's _UVic Engineering & Computer Science Students’ Society_. Should I change it back?

File: pages/index.tsx (hunk 2 above)
Comment (arfazca) [reply pages/index.tsx]: I like the second one! 

_VikeLabs is a collective of students who learn to build, deploy, and test software apps. We are a community of student developers, designers, and entrepreneurs who are passionate about designing software solutions for students and the UVic campus community._

**Comment by arfazca** [comment]
> A couple of suggestions/discussions in other comments that should be resovled first. And #99 should be completed first because this duplicates the changes.

I have updated the #99 and merged those changes in this pull request as well. After resolving #99, we can successfully merge #100.

File: pages/about.tsx (hunk 1 above)
Comment (AmandaGAnderson) [reply pages/about.tsx]: No? I was pointing out line 84 in the file says "UVic Engineering Student Society" and should also be updated if everything else referencing the ECSS is udpated. Could replace it with UVic ECSS or UVic Engineering & Computer Science Students’ Society, up to you.

Comment (arfazca) [reply pages/about.tsx]: I updated the name change in the latest commit for this pull request. Let me know if everything looks good!

**Review by AmandaGAnderson — APPROVED** [review]
Looks good!

**Pull request merged**